src/PrestaShopBundle/Controller/Admin/Configure/AdvancedParameters/ProfilesController.php

```diff
@@ -181,24 +189,23 @@ public function bulkDeleteAction(Request $request)
     /**
      * Get human readable error for exception.
      *
-     * @param ProfileException $e
-     *
-     * @return string Error message
+     * @return array
      */
-    protected function handleDeleteException(ProfileException $e)
+    protected function getErrorMessages()
```

Comment:
it's a BC: sadly you have to deprecate the old one and create a new one (why do we need this function to be protected?)

Reply:
it's not a BC, this controller should be part of 1.7.6. For `protected` method, i don't know, maybe to make it extendable if developers decide to extend this controller, either way, i can make it `private` if you want. :)

Reply:
do you mean that handleDeleteException function doesn't exists in 1.7.5.x branch?

Reply:
whole `ProfilesController` does not exist in 1.7.5.x branch